presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -90,6 +90,11 @@ public WindowNode(
         return ImmutableList.copyOf(concat(source.getOutputSymbols(), windowFunctions.keySet()));
     }
 
+    public Set<Symbol> getCreatedSymbols()
```

Comment:
I think `getWindowSymbols` is misleading.
We want to emphasize these symbols are introduced by the window node.
`WindowNode` puts on output all symbols from input + additional symbols which come from executed window functions.

I'd keep `getCreatedSymbols`.